packages/contracts/contracts/libraries/Types.sol

```diff
@@ -139,16 +141,37 @@ library Types {
 
     /**
      * @notice Struct representing a challenge.
+     *
+     * @custom:field turn       The current turn.
+     * @custom:field asserter   Address of the asserter.
+     * @custom:field challenger Address of the challenger.
+     * @custom:field segments   Array of the segment.
+     * @custom:field segStart   The L2 block number of the first segment.
+     * @custom:field segSize    The number of L2 blocks.
+     * @custom:field timeoutAt  Timeout timestamp of the next turn.
+     * @custom:field outputRoot The L2 output root to replace.
```

Comment:
```suggestion
     * @custom:field outputRoot The L2 output root to be replaced.
```

Reply:
I think `outputRoot` here means the new output root to replace the existing one.
```suggestion
     * @custom:field outputRoot The L2 output root to replace the existing one.
```